Control/Uart.h

```diff
@@ -0,0 +1,21 @@
+#ifndef Uart_h
+#define Uart_h
+
+#include "Arduino.h"
+
+class Uart {
+public:
+    Uart();
+    void initializeCommUart();
+    void update();
+    float getDistance();
+    float getAngle();
+
+private:
+    String inputString;
+    bool stringComplete;
+    float distance;
+    float angle;
```

Comment:
According to the convention, this member variables should be snake case with a final underscore: `input_string_`